src/Apps/CollectorProfile/Routes/Saves/Components/ArtworkListItemsList.tsx

```diff
@@ -0,0 +1,112 @@
+import { Shelf, SkeletonBox } from "@artsy/palette"
+import { ArtworkListItemFragmentContainer } from "Apps/CollectorProfile/Routes/Saves/Components/ArtworkListItem"
+import { FC, useRef } from "react"
+import { graphql, useLazyLoadQuery } from "react-relay"
+import { useRouter } from "System/Hooks/useRouter"
+import { extractNodes } from "Utils/extractNodes"
+import { ArtworkListItemsListQuery } from "__generated__/ArtworkListItemsListQuery.graphql"
+import { times } from "lodash"
+
+export const ArtworkListItemsList: FC = () => {
+  const { me } = useLazyLoadQuery<ArtworkListItemsListQuery>(query, {})
+
+  const {
+    match: { params },
+  } = useRouter()
+
+  const initialArtworkListId = useRef(params.id)
+
+  if (!me) {
+    return null
+  }
+
+  const savedArtworksArtworkList = me.savedArtworksArtworkList
```

Comment:
All this logic was straight extracted from [CollectorProvileSavesRoute](https://github.com/artsy/force/blob/b7c4dfe237bf7948428cf62df7fd7ec6f6ce90a5/src/Apps/CollectorProfile/Routes/Saves/CollectorProfileSavesRoute.tsx#L25) component - I don't want to refactor anything in this. PR